src/view/presenter.rs

```diff
@@ -92,27 +91,30 @@ impl<'p> Presenter<'p> {
 
         entries.iter().enumerate().fold(0, |offset, (index, element)| {
             let content = match entries.len() {
-                1 => {
-                    // There's only one element; have it fill the line.
-                    element.content.pad_to_width(self.view.terminal.width())
+                // There's onyl one element; have it fill the line.
+                1 => format!(
+                    "{:width$}",
+                    element.content,
+                    width=self.view.terminal.width(),
+                ),
+
+                // Expand the last element to fill the remaining width.
+                2 if index == entries.len() - 1 => format!(
+                    "{:width$}",
+                    element.content,
+                    width=self.view.terminal.width().saturating_sub(offset),
+                ),
+                2 => element.content.clone(),
```

Comment:
```suggestion
```

Pretty sure we can leave this to the default clause to catch.

Reply:
Actually, no, this might satisfy the match guard on the first wildcard match, so it should be kept here.